2017-bae-europe/eu_future/eu_future.ipynb

```diff
@@ -137,10 +133,164 @@
   },
   {
    "cell_type": "code",
+   "execution_count": 3,
+   "metadata": {},
+   "outputs": [
+    {
+     "name": "stdout",
+     "output_type": "stream",
+     "text": [
+      "Total Tails in MTHM:\n",
+      "1081826.9578\n"
+     ]
+    }
+   ],
+   "source": [
+    "# get tailings timeseries both cum and non-cum\n",
+    "cur = an.get_cursor('hope.sqlite')\n",
+    "\n",
+    "init_year, init_month, duration, timestep = an.get_timesteps(cur)\n",
+    "\n",
+    "agent_ids = an.get_agent_ids(cur, 'Sink')\n",
+    "tails_dict = an.facility_commodity_flux(cur, agent_ids, ['tails'], False)\n",
+    "tails_no_cum = an.facility_commodity_flux(cur, agent_ids, ['tails'], False, False)\n",
+    "an.double_axis_bar_line_plot(tails_no_cum, tails_dict, timestep, 'Years', 'Monthly Discharge Mass [MTHM]', 'Cumulative Mass [MTHM]',\n",
+    "                    'Tails Mass vs Time', 'tailings_cum_no_cum', init_year)\n",
```

Comment:
Why is this plot created but not displayed?

Reply:
Thanks for the comment! I realize that I have left all my trails of thought.. I will clean it up to a concise, clean report. Much appreciated